src/backend/commands/analyze.c

```diff
@@ -1436,7 +1431,7 @@ static Oid buildSampleTable(Oid relationOid,
 /**
  * Drops the sample table created during ANALYZE.
  */
-static void dropSampleTable(Oid sampleTableOid)
+void dropSampleTable(Oid sampleTableOid, bool isExternal)
```

Comment:
I'm confused here. The sample table we created, even for pxf, is a regular AO table. Why we use 'drop external table' here?